app/components/pipeline/modal/toggle-job/component.js

```diff
@@ -18,7 +18,7 @@ export default class PipelineModalToggleJobComponent extends Component {
   async updateJob() {
     const payload = {
       state:
-        this.args.toggleAction.toLowerCase() === 'disabled'
+        this.args.toggleAction.toLowerCase() === 'disable'
```

Comment:
should this still be disabled?
```suggestion
        this.args.toggleAction.toLowerCase() === 'disabled'
```

Reply:
It's actually supposed to be `disable`...it was a bit of a typo/bug when I had initially put it in.  The main idea is that the component takes in a verb and the UI changes it to the correct tense for display.

Reply:
okay I see